src/yetibot/core/handler.clj

```diff
@@ -52,6 +54,15 @@
        ; ensure prefix is actually a command
        (filter #(command? (-> % second second second)))))
 
+(defn has-command-prefix?
+  "Returns true if body has an command matching the prefix"
+  ([body]
+    (re-find #"^\!(.+)" body))
+  ([body prefix] 
+    (re-find (re-pattern (str "^\\" prefix "(.+)")) body)))
+
+(def ^:private config-prefix (:value (get-config sch/Str [:command :prefix])))
```

Comment:
Could bake the default in here in order to avoid multi-arity `has-command-prefix?` function, e.g.:

```clojure
(def ^:private config-prefix
  (or (:value (get-config sch/Str [:command :prefix])) "!"))
```